files/image_config/resolv-config/update-containers

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-for container in $(docker ps -a --format=" {{ .ID }}"); do
+networking_status=$(systemctl is-active networking.service 2>/dev/null)
```

Comment:
why do you need this change?

Reply:
This is a performance optimization. We don't need to update the containers if the networking is inactive. When the networking is back online, one more update will be triggered. This reduces fast/warm reboot time on devices with weak CPUs.